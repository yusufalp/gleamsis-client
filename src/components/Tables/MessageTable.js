import React from 'react';

export default function MessageTable() {
  return (
    <table>
      <thead>
        <tr>
          <th>From</th>
          <th>Subject</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Yusuf Alp</td>
          <td>Requesting purchase approval</td>
        </tr>
        <tr>
          <td>Thinkful</td>
          <td>Congratulations!</td>
        </tr>
      </tbody>
    </table>
  );
}