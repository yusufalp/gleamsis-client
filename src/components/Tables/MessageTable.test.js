import React from 'react';
import ReactDOM from 'react-dom';
import MessageTable from './MessageTable';
import renderer from 'react-test-renderer';

describe('MessageTable component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MessageTable />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<MessageTable />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});