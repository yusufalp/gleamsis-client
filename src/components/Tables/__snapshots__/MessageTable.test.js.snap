// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageTable component renders the UI as expected 1`] = `
<table>
  <thead>
    <tr>
      <th>
        From
      </th>
      <th>
        Subject
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        Yusuf Alp
      </td>
      <td>
        Requesting purchase approval
      </td>
    </tr>
    <tr>
      <td>
        Thinkful
      </td>
      <td>
        Congratulations!
      </td>
    </tr>
  </tbody>
</table>
`;
